CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
           #[[NOGDI]] # We need GDI for now, enable when GDI is actually not needed (or when all windows.h mentions are gone)
   )
 
-  list(APPEND CMAKE_LIBRARY_PATH "lib/win" "lib/win/directx")
+  list(APPEND CMAKE_LIBRARY_PATH "lib/win" "lib/win/directx/${CMAKE_CXX_COMPILER_ARCHITECTURE_ID}")
```

Comment:
Since there changes happens... Can you please change it to `lib/win/DirectX/...`?

Reply:
Good catch, done.